File: javascript/19_packages.js
// 19 packages / dependencies

// Node package managers:
//   npm install lodash
//   yarn add lodash
//   pnpm add lodash

// package.json holds dependencies
// node_modules/ is where they live

console.log("JS packaging:");
console.log("  npm / yarn / pnpm");
console.log("  package.json + node_modules");
console.log("  require() or import");

// built-in example
const path = require("path");
console.log("stdlib path.basename:", path.basename("/a/b/c.js"));

File: python/19_packages.py
# 19 packages / dependencies

# Python package managers:
#   pip install requests
#   poetry add requests
#   uv add requests

# requirements.txt example:
# requests==2.32.0
# pytest>=8.0

# pyproject.toml is the modern standard

print("Python packaging:")
print("  pip / poetry / uv")
print("  requirements.txt or pyproject.toml")
print("  import after install")

try:
    import json  # stdlib - always available
    print("stdlib json: ok")
except ImportError:
    pass
